src/server/stream_family_test.cc

```diff
@@ -499,4 +499,54 @@ TEST_F(StreamFamilyTest, XTrimInvalidArgs) {
   EXPECT_THAT(resp, ErrArg("syntax error"));
 }
 
+TEST_F(StreamFamilyTest, Xclaim) {
+  Run({"xadd", "foo", "1-0", "k1", "v1"});
+  Run({"xadd", "foo", "1-1", "k2", "v2"});
+  Run({"xadd", "foo", "1-2", "k3", "v3"});
+  Run({"xadd", "foo", "1-3", "k4", "v4"});
+
+  // create a group for foo stream
+  Run({"xgroup", "create", "foo", "group", "0"});
+  // alice consume all the stream entries
+  Run({"xreadgroup", "group", "group", "alice", "streams", "foo", ">"});
+
+  // bob claims alice's two pending stream entries
+  auto resp = Run({"xclaim", "foo", "group", "bob", "0", "1-2", "1-3"});
+  EXPECT_THAT(resp, RespArray(ElementsAre(RespArray(ElementsAre("1-2", ArrLen(2))),
```

Comment:
Here and below, any reason to use `ArrLen()` instead of outlining the actual values in the array? Shouldn't they be just `"k1"` and `"v1"`, etc?

Reply:
There is no particular reason. I basically followed other test cases where `ArrLen` is used :)